package.json

```diff
@@ -13,10 +13,10 @@
   },
   "dependencies": {
     "amqp": "0.2.6",
-    "async": "0.2.7",
-    "lodash": "4.17.5",
+    "async": "1.4.0",
```

Comment:
no breaking changes in our use of async, which is restricted to async.each() calls